shared/pingpong/accounts.go

```diff
@@ -288,7 +335,13 @@ func prepareAssets(accounts map[string]uint64, client libgoal.Client, cfg PpConf
 
 			assetAmt := assetParams[k].Total / uint64(len(accounts))
 			for addr2 := range accounts {
-				if addr != addr2 {
+				optedIn := false
```

Comment:
Why not `continue` here if `addr == addr2`?

Reply:
because it was written this way. I'll fix, it makes sence.